packages/console/src/App.tsx

```diff
@@ -31,6 +34,24 @@ const Main = () => {
   const location = useLocation();
   const navigate = useNavigate();
   const fetcher = useSwrFetcher();
+  const { data } = useSWR<Setting, RequestError>('/api/settings');
+  const defaultTheme = localStorage.getItem(themeStorageKey) ?? AppearanceMode.SyncWithSystem;
```

Comment:
this will get from localStorage on every render execution (which is way more times than the real render). shall we optimize?

Reply:
unlike `refreshToken` it's okay to fetch from the storage only at init

Reply:
Good point. I decided to put it in the outer scope of the class. And tested that it only called once on app init